pymongo/collection.py

```diff
@@ -1166,10 +1222,12 @@ def delete_many(self,
             self._delete_retryable(
                 filter, True,
                 write_concern=write_concern,
-                collation=collation, hint=hint, session=session, let=let),
+                collation=collation, hint=hint, session=session, let=let,
+                comment=comment),
             write_concern.acknowledged)
 
-    def find_one(self, filter: Optional[Any] = None, *args: Any, **kwargs: Any) -> Optional[_DocumentType]:
+    def find_one(self, filter: Optional[Any] = None, *args: Any, **kwargs: Any
+    ) -> Optional[_DocumentType]:
```

Comment:
These changes are not needed. I suppose they will be resolved by rebasing this PR after the black/isort changes.

Reply:
Done.